BUFRCREX_CodeFlag_en_42.csv

```diff
@@ -3,3 +3,9 @@ FXY,ElementName_en,CodeFigure,EntryName_en,EntryName_sub1_en,EntryName_sub2_en,N
 042004,Confidence of inversion for each partition of swell wave spectra,1,180-degree ambiguity not resolved,,,,,Operational
 042004,Confidence of inversion for each partition of swell wave spectra,2-14,Reserved,,,,,Operational
 042004,Confidence of inversion for each partition of swell wave spectra,15,Missing,,,,,Operational
+042017,Quality flag of swell wave spectra,0,Very good,,,,,Operational
+042017,Quality flag of swell wave spectra,1,Good,,,,,Operational
+042017,Quality flag of swell wave spectra,2,Medium,,,,,Operational
+042017,Quality flag of swell wave spectra,3,Low,,,,,Operational
+042017,Quality flag of swell wave spectra,4,Poor,,,,,Operational
+042017,Quality flag of swell wave spectra,15,Missing,,,,,Operational
```

Comment:
@marijanacrepulja Can you look at this question?

Reply:
@amilan17 
It should be 5-14, Reserved
15,Missing value

On the other note, we didnt have a chance to discuss this proposal with the team. Do you want to include it and we discuss it at the meeting in September. If there is no objections we will keep it otherwise withdraw it from FT2023-2 and move it to FT2024-1.